test/functional/test-log.js

```diff
@@ -382,43 +381,18 @@ describe('Logging', () => {
       expect(error.expected).to.be.true;
     });
 
-    it('should create suffixed errors from message', () => {
+    it('should create user errors from message', () => {
       const error = log.createError('test');
       expect(error.expected).to.be.undefined;
       expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
+      expect(isUserError(error)).to.be.true;
     });
 
-    it('should create suffixed errors from error', () => {
-      const error = log.createError(new Error('test'));
-      expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
-    });
-
-    it('should only add suffix once', () => {
-      const error = log.createError(new Error('test' + USER_ERROR_SENTINEL));
-      expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
-    });
-
-    it('should strip suffix if not available', () => {
-      const error = log.createError(new Error('test'));
-      expect(isUserErrorMessage(error.message)).to.be.true;
-
-      const noSuffixLog = new Log(win, RETURNS_FINE);
-      noSuffixLog.createError(error);
-      expect(isUserErrorMessage(error.message)).to.be.false;
-    });
-
-    it('should create other-suffixed errors', () => {
-      log = new Log(win, RETURNS_FINE, '-other');
```

Comment:
@dvoytenko Is there any reason you added this test case? Would there be other use to this suffix? If not, I've made some changes to use boolean instead of string (would be useful for my next step - checking error embed). What do you think?